omod/src/main/java/org/openmrs/module/aijar/page/controller/MarkPatientDeadPageController.java

```diff
@@ -41,14 +42,18 @@ public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcru
         }
     }
 
-    public String post(@RequestParam(value = "causeOfDeath",required = false) String causeOfDeath, @RequestParam(value = "dead",required = false) Boolean dead, @RequestParam(value = "deathDate",required = false) Date deathDate, @RequestParam("patientId") String patientId, UiUtils
-            uiUtils) {
+    public String post(@RequestParam(value = "causeOfDeath",required = false) String causeOfDeath, @RequestParam(value = "dead",required = false) Boolean dead, @RequestParam(value = "deathDate",required = false) Date deathDate, @RequestParam("patientId") String patientId) {
         PersonService personService = Context.getPersonService();
+
         try {
             PatientService patientService = Context.getPatientService();
             Patient patient = patientService.getPatientByUuid(patientId);
+
+
             Person person = personService.getPerson(patient.getPatientId());
```

Comment:
There are reasons why I fetch a person is to be able to use their UUID remember that patient uuid is different from person UUID